src/client/gui/lib/update_available.dart

```diff
@@ -1,7 +1,10 @@
+import 'dart:io';
 import 'package:basics/basics.dart';
 import 'package:flutter/material.dart';
+import 'package:local_notifier/local_notifier.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:flutter_svg/svg.dart';
+import 'package:multipass_gui/platform/platform.dart';
```

Comment:
for our own files, it's preferable you use relative imports e.g. `import 'platform/platform.dart'` just so it's separated from the 3rd party stuff